whisperv6/mailserver_response.go

```diff
@@ -40,22 +41,22 @@ func CreateMailServerRequestFailedPayload(requestID common.Hash, err error) []by
 // * request completed successfully
 // * request failed
 // If the payload is unknown/unparseable, it returns `nil`
-func CreateMailServerEvent(payload []byte) (*EnvelopeEvent, error) {
+func CreateMailServerEvent(peer enode.ID, payload []byte) (*EnvelopeEvent, error) {
```

Comment:
Small detail but we usually name variables `peer` if it's `Peer` struct. Here, we can use `nodeID` or `peerID` I think.